tools/docker/docker-compose-stage.yaml

```diff
@@ -75,6 +75,27 @@ services:
     volumes:
       - ${EDA_HOST_PODMAN_SOCKET_URL:-/run/user/1000/podman/podman.sock}:/run/podman/podman.sock:z
 
+  eda-manager:
+    user: "${EDA_POD_USER_ID:-0}"
+    deploy:
+      replicas: ${EDA_NUM_WORKERS:-1}
+    image: "${EDA_IMAGE:-quay.io/ansible/eda-server:main}"
+    environment: *common-env
+    command:
+      - aap-eda-manage
+      - rqworker
+      - --with-scheduler
```

Comment:
We must take care of this if we finally merge https://github.com/ansible/eda-server/pull/263
(now the PR link is broken, it will be fixed soon) It is the PR where is implemented the RQ scheduler, so the scheduler is executed in a different pod